go.mod

```diff
@@ -204,10 +172,8 @@ require (
 	golang.org/x/text v0.3.7 // indirect
 	golang.org/x/time v0.0.0-20210723032227-1f47c861a9ac // indirect
 	google.golang.org/protobuf v1.28.0 // indirect
-	gopkg.in/ini.v1 v1.66.2 // indirect
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0 // indirect
 	gopkg.in/natefinch/npipe.v2 v2.0.0-20160621034901-c1b8fa8bdcce // indirect
-	gopkg.in/yaml.v2 v2.4.0 // indirect
```

Comment:
34 fewer dependencies apparently :partying_face:

Reply:
Maybe just needed a tidy anyways? :shrug: Either way :tada:

Reply:
Yeah, quite possible! I don't remember when I did a `tidy` last time on this one :(